src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/Compiles.cs

```diff
@@ -439,8 +443,10 @@ public static IEnumerable<object[]> CustomCollections()
         }
 
         [Theory]
-        [MemberData(nameof(CodeSnippetsToCompile))]
-        [MemberData(nameof(CustomCollections))]
+        //[MemberData(nameof(CodeSnippetsToCompile))]
+        //[MemberData(nameof(CustomCollections))]
+        [MemberData(nameof(MyTests))]
+
```

Comment:
```suggestion
        [MemberData(nameof(CodeSnippetsToCompile))]
        [MemberData(nameof(CustomCollections))]
```

Reply:
Oops, forgot to change the tests back, thanks for pointing that out.

Reply:
I think this file might need to be reverted. I'm not terribly concerned with adding a new test here. My assumption on the test front would be that we should have observed a warning, no?

Reply:
What do you mean we should have observed a warning?

Reply:
It's an error from having a LibraryImport in a ref struct. I think it's just so uncommon put a LibraryImport in a ref struct before so we hadn't found the issue previously.